lib/Extension/LanguageServerCodeTransform/CodeAction/PromotePropertyProvider.php

```diff
@@ -36,7 +36,7 @@ public function provideActionsFor(TextDocumentItem $textDocument, Range $range,
 
         return new Success([
             new CodeAction(
```

Comment:
Use named parameters for protocol classes as argument order can change when it's regenerated

Reply:
Will do in the future, would you be opposed to abstracting this away and giving the `promoteProperty` Function on the handler an attribute that automatically registers it as a `CodeAction` provider?

Reply:
no - at least not without some thought. I think the current way is explicit, adding an attribute would be "implicit" while not actually adding on LOC - because assuming we would not automatically scan all the classes in the project to find out which ones have attributes, you'd still need to register the class in the DI with a tag anyway...

Reply:
I think you kinda get the point but my idea on the implementation would be a little different. I'll open up an issue explaining it further.